core/src/com/mygdx/game/client/service/TurnService.java

```diff
@@ -0,0 +1,12 @@
+package com.mygdx.game.client.service;
+
+import java.util.function.Function;
+
+public interface TurnService {
+  TurnToken peekToken();
+
+  void obtainToken(Function<TurnToken, Void> onSuccess,
```

Comment:
Tu potrzebuję pomysłu jaki interfejs można użyć, który bierze wartość, ale nie zwraca nic. W rxJava to chyba był Completable, ale tego tutaj nie ma. 

Jest jeszcze Flow z JDK9, ale go nie wiem jak się używa i rzeczy z reaktora, które będziemy używali w sieciowości jak będzie netty. Więc w sumie dopiero jak dobrze użyjemy netty to będzie widać jak powinny jednak wyglądać te asynchroniczne interfejsy tutaj.